src/index.scss

```diff
@@ -491,7 +493,6 @@ header {
 
 .comments-sort {
   margin-bottom: -44px;
-  z-index: 1;
```

Comment:
After removing the z-index. Please check the comment sort dropdown is clickable or not.